common/yak/yakurl/ssaurl_test.go

```diff
@@ -118,13 +118,13 @@ func TestSFURl(t *testing.T) {
 		res, err := SendURL(local, progID, "/", `
 		target2(* #-> as $a) 
 		target1() as $target1
-		alert $target1 for "alert information"
+		alert $target1 for {msg: 'alert information'}
```

Comment:
add bool field in `ExtraInfo`, this `alert $a for "msg"` just get "msg" not `{"Msg": "msg", ...}`.
this Field `OnlyMsg` can set in compiler(syntaxflow antlr visitor)(visit_statement.go:177)